doc/lua_api.txt

```diff
@@ -4327,6 +4328,7 @@ Call these functions only at load time!
       by default.
     * To allow players with `basic_privs` to grant, see the `basic_privs`
       minetest.conf setting.
+    * To modify an already registered privilege just edit the value found in minetest.registered_privileges
```

Comment:
```suggestion
    * Registered privileges can be modified directly in
      `minetest.registered_privileges`.
```

Reply:
Please also apply this wording in other places, it's better.